docs/changelog.rst

```diff
@@ -3,6 +3,13 @@ Changelog
 
 .. scriv-insert-here
 
+.._changelog-3.2.1
+
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Added Podman as a valid container type for the |GlobusComputeEngine|.
+
```

Comment:
We use `scriv` to create changelogs, as it will manage details automatically.  For example, we won't be releasing will likely not release a 3.2.1 version.  This is (simplistically) described in `changelog.d/README.md`.

Please reach out if that is not enough of a pointer.  (I know you're working with Ryan already, and can also ping me on Slack if needs be.)  Thanks!